src/d3a/models/strategy/__init__.py

```diff
@@ -397,18 +397,31 @@ def can_bid_be_posted(self, bid_energy, required_energy_kWh, market):
         posted_energy = (bid_energy + self.posted_bid_energy(market.id))
         return posted_energy <= required_energy_kWh
 
+    def is_bid_posted(self, market, bid_id):
+        return bid_id in [bid.id for bid in self.get_posted_bids(market)]
+
     def posted_bid_energy(self, market_id):
```

Comment:
Out of context of this PR but I just realised that this method should have a better name like `posted_bid_cumulative_energy `

Reply:
Hmm, I am not convinced. But if @spyrostz agrees, will rename it.

Reply:
I also disagree. I think the name is concise and quite understandable. `Cumulative` is a term that I , in general, do not like the way we use it, because it implies that there is a reduce function, or that more results will be added in subsequent steps. I think `accumulated` or even better `sum`, is the correct term when we want to display the final result of an addition. But for this case, the function name is clear (to my eyes at least): energy of the posted bids, therefore no need to rename that.